worldengine/draw.py

```diff
@@ -129,6 +210,109 @@ def elevation_color(elevation, sea_level=1.0):
     return _sature_color(_elevation_color(elevation, sea_level))
 
 
+def get_normalized_elevation_mask(world):
```

Comment:
This doesn't return a mask.
Also, do you actually need to differentiate between land and ocean like this? If not, this method could be done in a single line. numpy.interp(world.elevation['data'], [world.elevation['data'].min(), world.elevation['data'].max()], [0, 255])